shared_model/validators/field_validator.hpp

```diff
@@ -35,11 +35,13 @@ namespace shared_model {
      */
     class FieldValidator {
      public:
-      FieldValidator()
+      FieldValidator(unsigned long long future_gap = std::chrono::minutes(5)
```

Comment:
Also the constant (afaik can be even constexpr) can be extracted to the project-wide scope, e.g in _time.hpp_, not really sure

Reply:
Well, yeah
time.hpp not really suitable, but still that constant can be useful (e.g for client)